### ISSUE TITLE
[Package Issue]: Microsoft.PowerBI

### ISSUE THREAD

**Issue by ChristianVerhoeven** [microsoft/winget-pkgs#89954]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

Hash for package is not correct.

### Steps to reproduce

Use WinGet Install Microsoft.PowerBI, it will throw an error saying the hashes do not match

### Actual behavior

Use WinGet Install Microsoft.PowerBI, it will throw an error saying the hashes do not match

### Expected behavior

Hashes match after download

### Environment

```shell
Windows Package Manager v1.3.2691
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22000.1219
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2691.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by jedieaston** [comment]
Should be fixed once that PR merges. Let me know if it isn't :)

**Comment by Trenly** [comment]
Close with reason: Resolved by #89929;
